mmdet3d/ops/norm.py

```diff
@@ -42,10 +43,14 @@ class NaiveSyncBatchNorm1d(nn.BatchNorm1d):
         It is slower than `nn.SyncBatchNorm`.
     """
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fp16_enabled = False
+
+    @force_fp32(out_fp16=True)
```

Comment:
All BN layers will be forced to use fp32 and there is no need to add this line.

Reply:
This should be kept because in high version Pytorch the input of customized norm layers is FP16 but will not be converted due to [this line](https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/fp16_utils.py#L275).